### PR TITLE
Support numeric font weights #675

### PR DESCRIPTION
**Pull request by lquast**

### CODE REVIEW

**Comment by honsa** [comment]
Great, just a small step, but big thanks you for your effort 💯

**Comment by bsweeney** [comment]
Thanks for the PR. This should give me enough motivation to do the work for 0.8.4.

**Comment by Sattevelt** [comment]
👍  Well done. I just wrote a lengthy comment to a related bugreport to help pinpoint where the problem is. And it instantly became redundant once i saw your PR 😄. I haven't tested the PR but, the logic in there seems ok and it nicely solves the problem of what numeric weights to concider as 'bold'.

File: src/Css/Style.php [hunk 1]
```diff
@@ -929,31 +929,17 @@ function get_font_family()
...
-            if ($weight < 600) {
-                $weight = "normal";
-            } else {
-                $weight = "bold";
-            }
-        } else if ($weight === "bold" || $weight === "bolder") {
-            $weight = "bold";
+        if ($weight === 'bold') {
+            $weight = 700;
+        } elseif (preg_match('/^[0-9]+$/', $match[0])) {
```
[the remark this answers is not in the document]
Comment (lquast) [reply src/Css/Style.php]: Yes

**Pull request merged**